README.md

```diff
@@ -1 +1,161 @@
+```
+
+## process.cwd() 
+
+When it comes to `process.cwd()` we get the current working directory of the process. In other words, it's an absolute path to the location from which we launch our script (the script which has the `process.cwd()`). 
+
+In some cases, `process.cwd()` is the same as `__dirname`. It's when we launch the script from the folder in which the script is located, e.g., with `$ node customers.js` the value returned by `cwd()` is **the same** as the value of `__dirname`. 
+
+In any other location, the `cwd()` will be different from the `__dirname` value, e.g., with `$ node node-modules-path/node customers.js`, the value returned by `cwd()` is **different** from the value of `__dirname`.
+
+Knowing the difference between `cwd()` and `__dirname`, which one to choice when? My rule of thumb is:
+
+* Use `__dirname` when working with resources (files, modules, folders) when they are related to your script (e.g., one module), and you know that the relative path from your script is not going to change. For example, if you are creating a module, and you want to include a file in that module, you know the relative path to the file will stay the same because you're the creator of the module. Typically developers use `__dirname` when they write applications, and dependency modules.
+* Use `process.cwd()` when working with a directory unrelated to your code. For example, if you are creating a command-line tool which generates a boilerplate code, you'll want to be able to use the tool in any directory to create new directories and files relative to the location in which you launch the tool and not in the location of the tool. Typically developers use `process.cwd()` for command-line tools.
+
+## process.env.PWD
+
+The `process.env.PWD` is similar to the `process.cwd()` in the value of the absolute path. It will return the value of the directory from which you launched the Node process. PWD stands for Print Working Directory. For example, with `$ node node-modules-path/node customers`, the `PWD` variable will be ` /Users/azat/Documents/Code/learn-co` the same as the value of `process.cwd().
+
+
+However, there are a few differences between `PWD` and `cwd()`. `PWD` is an environment variable and not a function. `PWD` is a call to the command on Unix-like systems:
+
+```
+$ pwd
+```
+
+You can also print the environment variable from the command line with `echo`:
+
+```
+$ echo $PWD
+```
+
+Another difference is that `PWD` can be changed with an assignment, just like any environment variable, while the current working directory can be changed with `process.chdir()`. Changing `PWD` won't affect the value of `cwd()`.
+
+Also, the `PWD` env var is a POSIX (Unix, Linux, Mac OS X) variable which mean it won't work on Windows. The `process.cwd()` will work on Windows.
+
+The `PWD` might be useful on POSIX systems (Unix, Linux, Mac OS X) to get the original value of the current working directory of the process. 
+
+
+## Importing a Folder 
```

Comment:
this seems to jump topics - can we add a transition to `PWD` and `cwd()` ? is there a transition?